Lib/configparser.py

```diff
@@ -300,10 +300,10 @@ def __init__(self, source=None, filename=None):
         # Exactly one of `source'/`filename' arguments has to be given.
         # `filename' kept for compatibility.
         if filename and source:
-            raise ValueError("Cannot specify both `filename' and `source'. "
-                             "Use `source'.")
+            raise ValueError("Cannot specify both 'filename' and 'source'. "
```

Comment:
It seems like a really odd quotation style, though, it isn't used elsewhere in `configparser.py`. Should I remove it nonetheless?